docs/documentation/stories/moving-into-the-cli.md

```diff
@@ -0,0 +1,63 @@
+# Moving your project to Angular-CLI
+
+It's easy to move your existing project to use Angular-CLI.
+The most straightforward way is to to make a new CLI project and copy over your app files.
+
+Let's see how to do it step by step.
+Here we use a project made with the [official QuickStart](https://github.com/angular/quickstart)
+as an example, but you should be able to adjust these instructions to other setups.
+
+Note for Windows users: we show unix commands here like `cp` to copy and `rm -rf` to delete files.
+Windows does not have these commands so use Explorer instead.
+
+Start with preparing your existing project folder, let's call it `awesome-app`.
+- commit and push your existing changes.
+- clean your folder from temporary files and ignored files using `git clean -fdx`.
+- rename your project folder to `old-awesome-app`.
+
+Now make a new project on the same folder as `old-awesome-app` using Angular-CLI.
```

Comment:
This could be un desirable if the person wants to keep their existing git integrations. Just keep that in mind.